pkg/python/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "gluesql-py"
+authors = ["Taehoon Moon <taehoon.moon@outlook.com>"]
```

Comment:
you can become the first author here :)
ref.
https://github.com/gluesql/gluesql/blob/bac635e489dcf617a4f47a752ab94f6470731f71/storages/json-storage/Cargo.toml#L3-L6

Reply:
Thanks!